### ISSUE TITLE
Build a method for cross-cluster promotion via skopeo

### ISSUE THREAD

**Issue by etsauer** [redhat-cop/pipeline-library#14]
@redhat-cop/developer-workflow

**Comment by etsauer** [comment]
```
          openshift.withCluster() {

            def localToken = readFile('/var/run/secrets/kubernetes.io/serviceaccount/token').trim()

            def secretData = openshift.selector('secret/prod-credentials').object().data
            def encodedRegistry = secretData.registry
            def encodedToken = secretData.token
            def registry = sh(script:"set +x; echo ${encodedRegistry} | base64 --decode", returnStdout: true)
            def token = sh(script:"set +x; echo ${encodedToken} | base64 --decode", returnStdout: true)

            openshift.withProject("${STAGE2}") {
              def imageRegistry = openshift.selector( 'is', "${APP_NAME}").object().status.dockerImageRepository
              echo "Promoting ${imageRegistry} -> ${registry}/${STAGE3}/${APP_NAME}"
              sh """
              set +x
              skopeo copy --remove-signatures \
                --src-creds openshift:${localToken} --src-cert-dir=/run/secrets/kubernetes.io/serviceaccount/ \
                --dest-creds openshift:${token}  --dest-tls-verify=false \
                docker://${imageRegistry} docker://${registry}/${STAGE3}/${APP_NAME}
              """
            }

          }
```

**Comment by pabrahamsson** [comment]
Fixed by #16
